### PR TITLE
add weekly releases for 2018-12-05

### PR DESCRIPTION
**Pull request by josephholsten**

### CODE REVIEW

File: content/posts/2018-12-05-releases.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- The build_essential resource is now directly used so that the built in resource in Chef 14+ can be used. This increases the required build-essential cookbook to 5.0+
+- Duplicate dependencies in the Berksfile have been removed
+- The chefignore file has been updated to prevent more unnecessary files from being uploaded to the chef server
+- Data bags are now loaded with the data_bag_item helper instead of Chef::EncryptedDataBagItem.load directly
+- Testing has been updated to use Delivery Local Mode which is built into ChefDK. The legacy Rakefile, Vagrantfile, Thorfile, and Cheffile have been removed
+- Platforms in Test Kitchen configurations have been updated and dokken images are now used in dokken
+
+## [`sc-chruby`](https://supermarket.chef.io/cookbooks/sc-chruby) - initial release!
+
+No really, that's all there is to be said.
```
Comment (tas50) [inline content/posts/2018-12-05-releases.md]: What does this cookbook even do?

Comment (josephholsten) [reply content/posts/2018-12-05-releases.md]: fair point. also, it probably could use some real docs @damacus. What would you like me to say about it?

Comment (josephholsten) [reply content/posts/2018-12-05-releases.md]: at least there's the `install` resource: https://github.com/sous-chefs/sc-chruby/blob/master/resources/install.rb

**Review by tas50 — APPROVED** [review]

**Comment by damacus** [comment]
We need a link to this page somewhere in the side navigation bar

**Pull request merged**